src/sysusers/sysusers.c

```diff
@@ -1785,27 +1861,33 @@ int main(int argc, char *argv[]) {
 
         if (optind < argc) {
                 int j;
-
-                for (j = optind; j < argc; j++) {
-                        k = read_config_file(argv[j], false);
-                        if (k < 0 && r == 0)
-                                r = k;
+                bool found_override = false;
+
+                if (arg_config_name) {
```

Comment:
I may misunderstand the logic, but seems that `arg_config_name` is ignored when no positional argument is provided. Right?